test/common_plat/validation/api/queue/queue.c

```diff
@@ -274,13 +275,18 @@ void queue_test_info(void)
 	CU_ASSERT(odp_queue_context_set(q_plain, q_plain_ctx,
 					sizeof(q_plain_ctx)) == 0);
 
+	memset(&capability, 0, sizeof(odp_queue_capability_t));
```

Comment:
This is benign but unnecessary as the parameter is an output variable to odp_queue_capability() and is initialized by the API itself.